sfdx-project.json

```diff
@@ -65,6 +65,14 @@
             "versionDescription": "TODO",
             "default": false
         },
+        {
+            "default": false,
+            "package": "Nebula Logger - Plugin - Async Failure Additions",
```

Comment:
After we merge your PR into my PR, I'll create this package in logger's dev hub & get it added to `sfdx-project.json`